charts/mimir-distributed/values.yaml

```diff
@@ -32,7 +32,7 @@ global:
 
 serviceAccount:
   create: true
-  name:
+  name: &serviceAccountName "mimir-service-account"
```

Comment:
The only thing I dislike is that the `nameOverride` and `fullnameOverride` logic doesn't work for the serviceAccount.name anymore.

Furthermore, when you set `serviceAccount.name` in your values, it doesn't update the `memcached.serviceAccount.name` YAML anchors below. Since the anchor doesn't exist in my values.yaml.

But I don't have a better solution, since this is a limitation of helm when you use sub-charts.

Reply:
Yeah, I'm not proud of the anchor thing :( , but that was the only thing I saw at the moment. 
I talked to my PM and and they are ok with having this feature in the next major version where we can break more things. If you can manage with this experimental chart in the mean time (couple of months) I'd rather defer merging it.